IdentityCore/src/workplacejoin/MSIDPkeyAuthHelper.m

```diff
@@ -30,6 +30,7 @@
 #import "MSIDJWTHelper.h"
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->